test/puppetlabs/trapperkeeper/services/authorization/authorization_service_test.clj

```diff
@@ -51,66 +51,60 @@
   [{:match-request
     {:path "/puppet/v3/catalog/([^/]+)"
      :type "regex"
-     :method :get}
+     :method "get"}
```

Comment:
Yeah, that sounds right to me.  Better to start with strings since that's what the raw input parsed from HOCON would look like.